bigbluebutton-html5/imports/ui/components/audio/audio-test/container.jsx

```diff
@@ -1,24 +1,31 @@
-import React, { Component } from 'react';
+import React from 'react';
 import { createContainer } from 'meteor/react-meteor-data';
+import Service from '/imports/ui/components/audio/service';
+import PropTypes from 'prop-types';
 import AudioTest from './component';
 
-class AudioTestContainer extends Component {
-  constructor(props) {
-    super(props);
-  }
+const propTypes = {
+  children: PropTypes.node,
+};
 
-  render() {
-    return (
-      <AudioTest {...this.props}>
-        {this.props.children}
-      </AudioTest>
-    );
-  }
-}
+const defaultProps = {
+  children: null,
+};
+
+const AudioTestContainer = props => (
+  <AudioTest {...props}>
```

Comment:
Wait, don't you map children here as well?
I think it should be something like:
```
const AudioTestContainer = ({ children, ...props }) => (
  <AudioTest {...props}>
    {children}
  </AudioTest>
...
```

Reply:
better yet, we could do it like this:
```jsx
const AudioTestContainer = props => <AudioTest {...props} />;
```